themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -12,13 +12,16 @@ function linkCallnumbers(callnumber, callnumber_handler) {
   return callnumber;
 }
 function displayItemStatus(result, $item) {
+   console.log("displayItemStatus"+result.full_status);
```

Comment:
I assume you will eventually want to remove these debug statements.

Reply:
Sure I'll clean this up including removing comments and indentation etc pending what we decide about this file in general.